python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -62,6 +63,43 @@ def test_ufunc_cudf_series_cupy_array(np_ar_tup, func):
 )
 def test_error_with_null_cudf_series(func):
     s_1 = cudf.Series([1, 2])
-    s_2 = cudf.Series([1, 2, None])
+    s_2 = cudf.Series([1, None])
+
+    # this thows a value error
+    # because of nulls in cudf.Series
     with pytest.raises(ValueError):
```

Comment:
So this is because `pd.Series([1, 2])` is of the type `float64` while  `cudf.Series([1,2])` if of `int64` and cupy/numpy does not support `Null` in non float types. If  we cast it appropriately like below it works:  
```python
>>> s1 = cudf.Series([1, 2])
>>> s2 = cudf.Series([1, None], dtype=np.float64).fillna(np.nan)
>>> np.less_equal(s1, s2)
0     True
1    False
dtype: bool
```

Reply:
I wonder if we should reroute the operations to cudf binops if we see cupy non-compatible types (e.g. nullable ints). But that's a discussion outside the scope of this PR.

Reply:
Yeah, so the workflow currently is : 

1. Look at `cudf` namespace for the presence of a function
2. Look at `cudf.Series` namespace for the presence of a function

If both these fail then only we will dispatch to cupy so re-routing can happen if we just add an alias for these ops in any of those name-spaces.  

See below: 
https://github.com/rapidsai/cudf/blob/bd537b6fbee34d5ab0fd1797d4397d82f8f31e0f/python/cudf/cudf/core/series.py#L789-L790

Reply:
Great, thanks for the explanation!